.ort.yml

```diff
@@ -1,9 +1,21 @@
 curations:
   license_findings:
+  - path: "analyzer/src/funTest/assets/projects/synthetic/php-composer/{empty-des, lockfile, no-deps, with-provide, with-replace}/composer.phar"
+    comment: "This file 'composer.phar' is part of PHP Composer and includes a mapping from human readable strings to SPDX license ids."
```

Comment:
Repeating "composer.phar" is redundant here.

Reply:
Done.